.github/actions/setup-docker-dashboard/action.yml

```diff
@@ -44,6 +39,11 @@ runs:
         };
         EOF
       shell: bash
+    - name: Build Juju Dashboard image
```

Comment:
I realised it needed to build the image _after_ modifying the config, so the config is included in the image.

Reply:
@Ninfa-Jeon do you mean the new `build.sh` scripts? If so, those are for building the charms, whereas this step builds a docker image that serves the dashboard (instead of being served by the charm). This image will be used by the JIMM team to run the dashboard within their docker compose env. Or am I missing something?